common/authorization/interceptor.go

```diff
@@ -50,6 +50,11 @@ type (
 	JWTAudienceMapper interface {
 		Audience(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo) string
 	}
+
+	// PostAuthorizationResponseProcessor processes the response in place after the request has been authorized.
+	PostAuthorizationResponseProcessor interface {
```

Comment:
nit: PostAuthorizationResponseProcessor -> PostAuthorizationAction

Reply:
Renamed to PostAuthorizationActionFn